File: Modul 4/Soal 4/PRAK404-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>
int main() {
    int pilihan;
    float a, b, c;

    while (1) {
        printf("Pilih Program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. EXIT\n");
        
        printf("Masukkan Pilihan: ");
        scanf("%d", &pilihan);

        if (pilihan == 1) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &a);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &b);
            c = a + b;
            printf("Hasil Penjumlahan antara %.2f dan %.2f adalah %.2f\n", a, b, c);
        } else if (pilihan == 2) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &a);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &b);
            c = a - b;
            printf("Hasil Pengurangan antara %.2f dan %.2f adalah %.2f\n", a, b, c);
        } else if (pilihan == 3) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &a);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &b);
            c = a * b;
            printf("Hasil Perkalian antara %.2f dan %.2f adalah %.2f\n", a, b, c);
        } else if (pilihan == 4) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &a);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &b);
            c = a / b;
            printf("Hasil Pembagian antara %.2f dan %.2f adalah %.2f\n", a, b, c);
        } else if (pilihan == 5) {
            printf("Terimakasih telah menggunakan kalkulator Anggraeni.\n");
            break;  
        } else {
            printf("Input anda salah, silahkan coba lagi.\n");
        }
    }

    return 0;
}

File: Modul 4/Soal 4/PRAK404-2410817220018-ANGGRAENI DWI ZAHRA.py
while(True):
    print("Pilih Program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. EXIT")
    j = int(input("Masukkan Pilihan :"))

    if j==1:
        a = int(input("Masukkan nilai pertama : "))
        b = int(input("Masukkan nilai kedua : "))
        c = (a + b)
        print(f"Hasil Penjumlahan antara {a:.2f} dengan {b:.2f} adalah {c:.2f}")
    elif j==2:
        a = int(input("Masukkan nilai pertama : "))
        b = int(input("Masukkan nilai kedua : "))
        c = (a - b)
        print(f"Hasil Pengurangan antara {a:.2f} dengan {b:.2f} adalah {c:.2f}")
    elif j==3:
        a = int(input("Masukkan nilai pertama : "))
        b = int(input("Masukkan nilai kedua : "))
        c = (a * b)
        print(f"Hasil Perkalian antara {a:.2f} dengan {b:.2f} adalah {c:.2f}")
    elif j==4:
        a = int(input("Masukkan nilai pertama : "))
        b = int(input("Masukkan nilai kedua : "))
        c = (a / b)
        print(f"Hasil Penjumlahan antara {a:.2f} dengan {b:.2f} adalah {c:.2f}")
    elif j==5:
        print("Terimakasih telah menggunakan kalkulator Anggraeni")
        break
    else:
        print("Input anda salah, silahkan coba lagi")